### DISCUSSION TITLE
Error in console when entering WPGraphQL

### DISCUSSION THREAD

**Discussion by mateusz-wyka** [wp-graphql/wp-graphql#2679] [General]
Hi

After WPGraphQL installation and entering the plugin in the admin I have error in console.
This error tells me nothing and I don't know what to do about it. Any ideas?

![obraz](https://user-images.githubusercontent.com/89961396/208097716-793e2270-d410-4a46-b632-88280a578be7.png)

**Comment by justlevine** [comment]
Follow the steps here: https://www.wpgraphql.com/docs/debugging#unexpected-token--in-json-at-position-0
